src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -105,6 +105,39 @@ export class SignifyApi {
     }
   }
 
+  async createDelegationIdentifier(delegatorPrefix: string): Promise<string> {
+    const signifyName = utils.uuid();
+    const icpResult2 = await this.signifyClient
+      .identifiers()
+      .create(signifyName, { delpre: delegatorPrefix });
+    const operation = await icpResult2.op();
```

Comment:
Does this always complete straight away, or can it fail?

Reply:
Basically, we wait the operation done to check it status, but for this case, the operation is false util the delegator accepted.
So I presume it always complete unless it throw an error

Reply:
OK, so if the delegator does not accept, it is always going to be false? Since this is async, then we should not wait for this operation.

Reply:
Correct, like creating multi-sig, and the rotation in the feature

Reply:
OK, in this case we shouldn't bother waiting - we should return after `.create()` I think